source/NVDAObjects/window/excel.py

```diff
@@ -229,13 +229,13 @@ class ExcelChartQuickNavItem(ExcelQuickNavItem):
 
 	def __init__( self , nodeType , document , chartObject , chartCollection ):
 		self.chartIndex = chartObject.Index
-		if chartObject.Chart.HasTitle:
+		if chartObject.Chart.HasTitle:
 
-			self.label = chartObject.Chart.ChartTitle.Text + " " + chartObject.TopLeftCell.address(False,False,1,False) + "-" + chartObject.BottomRightCell.address(False,False,1,False) 
+			self.label = chartObject.Chart.ChartTitle.Text + " " + chartObject.TopLeftCell.address(False,False,1,False) + "-" + chartObject.BottomRightCell.address(False,False,1,False) 

```

Comment:
Do you know why this ended up in the diff?

Reply:
yes, there is a commit that converts lines in that file to windows lines. It is the last commit, you can exclude it from the diff.